_docs_integrate/requests-via-relationshiptemplates.md

```diff
@@ -56,77 +65,93 @@ For an overview of the available [RequestItems]({% link _docs_integrate/data-mod
+However, if an active Relationship already exists, the Recipient receives the Request specified in the `onExistingRelationship` property when loading the RelationshipTemplate, if such is specified. The specification of a Request in the `onNewRelationship` property is mandatory in contrast to the specification of a Request in the `onExistingRelationship` property.
+
+Note that the same Request can be specified in the `onExistingRelationship` property as in the `onNewRelationship` property, but a different Request can also be used.
+These customization options are useful as the Recipient that will load the RelationshipTemplate may not be known in advance.
+It is therefore possible that a Relationship between the Sender and the Recipient already exists or not.
+It is also possible to configure the RelationshipTemplate so that it can be loaded by multiple Identities.
+The creator of the RelationshipTemplate may have a Relationship to some of these Identities and no Relationship to others of these Identities.
+{: .notice--info}
+
+To create a RelationshipTemplate, the instructions of the [Create own RelationshipTemplate]({% link _docs_use-cases/use-case-transport-create-own-relationshiptemplate.md %}) use case documentation must be followed.
+A RelationshipTemplateContent needs to be specified in the `content` of the payload because the Sender wants to send the Recipient a Request via the RelationshipTemplate.
+In the payload example below, the [Request whose validity we have already checked](#check-the-requests-validity) is contained both within the `onNewRelationship` property and within the `onExistingRelationship` property of the [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent).
 
 ```jsonc
 {
   "maxNumberOfAllocations": 1,
-  "expiresAt": "2025-01-01T00:00:00.000Z",
+  "expiresAt": "2026-01-01T00:00:00.000Z",
   "content": {
+    // RelationshipTemplateContent
     "@type": "RelationshipTemplateContent",
-    "title": "Requests via RelationshipTemplates example",
     "onNewRelationship": {
-      // the content property of the payload of the previous step,
-      // if the Connectors don't have a Relationship with each other, yet
-      ...
+      // Specification of a Request
+      "items": [
+        {
+          "@type": "AuthenticationRequestItem",
+          "mustBeAccepted": true,
+          "title": "The Sender is asking for an authentication."
+        }
+      ]
     },
     "onExistingRelationship": {
-      // the content property of the payload of the previous step,
-      // if the Connectors already have a Relationship with each other
-      ...
+      // Specification of another Request
+      "items": [
+        {
+          "@type": "AuthenticationRequestItem",
+          "mustBeAccepted": true,
+          "title": "The Sender is asking for an authentication."
+        }
+      ]
     }
   }
```

Comment:
I liked it more as it was before, b/c the reader understands that he has to input **exactly the same as before**

Reply:
To make this clear, I've added in the text right before the example the sentence "In the payload example below, the Request whose validity we have already checked is contained both within the `onNewRelationship` property and within the `onExistingRelationship` property of the RelationshipTemplateContent."

Is that clear enough, or would you like an alternative phrasing?

Reply:
for me it's not about the phrasing. Tutorial readers skip text. They will only read text when required - having an incomplete payload makes them read more text.